### PR TITLE
hover effect on opportunity box

### PR DESCRIPTION
**Pull request by kumarishalini6**
fixes#issue-249

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
I have gave some background color , border color and hover effect.
Opportunity Calendar - Personal - Microsoft​ Edge 5_12_2021 01_29_15 PM

### CODE REVIEW

**Comment by kumarishalini6** [comment]
video:-

https://user-images.githubusercontent.com/69043944/117940763-e07c4f80-b2b5-11eb-84e6-37f351d8f16b.mp4

File: src/CSS/Opportunity.module.css [hunk 1]
```diff
@@ -16,7 +16,17 @@
...
+    box-shadow: 0px 12px 12px -10px #ccc;
+    transition: all 0.6s;
+    border-color: gray;
+    background-color: #FBFCFC ; 
+    
+}
+.Card:hover {
+    z-index: 99;
+    box-shadow: 2px 2px 30px #b9b9b9;
+    transform: scale(1.2);
```
Comment (vaishali614) [inline src/CSS/Opportunity.module.css]: Change it to transform: scale(1.05);

File: src/CSS/Opportunity.module.css [hunk 2]
```diff
@@ -16,7 +16,17 @@
     margin-top: 3rem;
     border-radius: 15px;
     border-radius: 15px;
-    box-shadow: 0px 2px 2px 0px #ccc;
+    box-shadow: 0px 12px 12px -10px #ccc;
+    transition: all 0.6s;
+    border-color: gray;
```
Comment (vaishali614) [inline src/CSS/Opportunity.module.css]: Remove border-color

**Review by vaishali614 — CHANGES_REQUESTED** [review]

File: src/CSS/Opportunity.module.css (hunk 1 above)
Comment (kumarishalini6) [reply src/CSS/Opportunity.module.css]: Ok

File: src/CSS/Opportunity.module.css (hunk 2 above)
Comment (kumarishalini6) [reply src/CSS/Opportunity.module.css]: Ok

**Comment by kumarishalini6** [comment]
done

**Pull request merged**